nebula/ui/components/MZRadioDelegate.qml

```diff
@@ -13,6 +13,7 @@ RadioDelegate {
     property bool isHoverable: true
     property var radioButtonLabelText
     property string accessibleName
+    property bool isRadioButtonLabelAccessible: true
```

Comment:
Seems like a good spot to use an alias since we are just essentially exporting/exposing the `Accessible.ignored` property of the label

Reply:
Disregard this comment - you cannot alias to an attached property (like `Accessible.ignored`)